src/direction.rs

```diff
@@ -55,7 +55,7 @@ impl Direction {
 
 impl Distribution<Direction> for Standard {
     fn sample<R: Rng + ?Sized>(&self, rng: &mut R) -> Direction {
-        match rng.gen_range(0, 4) {
+        match rng.gen_range(0..=4) {
```

Comment:
Post-Morten comment. This change makes the `unreachable` reachable. Fixed in: https://github.com/lemunozm/asciiarena/commit/f88a77b0e5a5c1fc28d83ad88763658a4cb79d3a